docs/debugging/debugging.md

```diff
@@ -0,0 +1,205 @@
+# Debugging
+
+This section describes procedures which may be used to debug
+`ropensci-review-bot` problems. The most common problem is a failure of the bot
+to respond. This section describes how to diagnose and resolve that problem,
+with solutions to other problems also generally found by following similar
+procedures. 
+
+## The Heroku service
+
+The bot itself is hosted on Heroku, which is the first point of diagnosis. The
+logs for the bot will reveal whether commands were successfully received. These
+logs can be viewed by logging in to [heroku](https://heroku.com), clicking on
+"radiant-garden", then under "More" on the upper right, clicking "Logs".
```

Comment:
I think in reality one needs to use the CLI to view past logs. I only ever see the current log through the interface but in that case I relaunch the command in GitHub once I opened the log window.

Reply:
Can you please then suggest an improvement for that statement? Thanks!